adobe/react-spectrum
# Flex should now render a passed id

Closes #1010 

## ✅ Pull Request Checklist:


## 📝 Test Instructions:

1. Create a Flex component with an id.
2. Check that the Flex is rendered with the id passed

## 🧢 Your Project:
